### PR TITLE
LPS-170047 Created "Retunm to Open" actions according to the status "COMPLETE"

### PR DESCRIPTION
**Pull request by rafaelfs98**
[LPS-164983](https://issues.liferay.com/browse/LPS-164983)

### CODE REVIEW

File: modules/apps/site-initializer/site-initializer-testray/site-initializer-testray/extra/remote-app/src/pages/Testflow/Subtask/useSubtasksActions.tsx [hunk 1]
```diff
@@ -31,6 +33,30 @@ const useSubtasksActions = () => {
...
+					updateItemFromList(
+						mutate,
+						0,
+						{},
+						{
+							revalidate: true,
+						}
+					);
+				});
+			},
```
Comment (welitonpe) [inline modules/apps/site-initializer/site-initializer-testray/site-initializer-testray/extra/remote-app/src/pages/Testflow/Subtask/useSubtasksActions.tsx:48]: Suggestion: you can simplify returning directly

```suggestion
		{
			action: (subtask, mutate) => 
				testraySubTaskImpl.assignToMe(subtask).then(() => 
					updateItemFromList(
						mutate,
						0,
						{},
						{
							revalidate: true,
						}
					)
				),
```

File: modules/apps/site-initializer/site-initializer-testray/site-initializer-testray/extra/remote-app/src/pages/Testflow/Subtask/useSubtasksActions.tsx [hunk 2]
```diff
@@ -65,13 +91,45 @@ const useSubtasksActions = () => {
...
```
Comment (welitonpe) [inline modules/apps/site-initializer/site-initializer-testray/site-initializer-testray/extra/remote-app/src/pages/Testflow/Subtask/useSubtasksActions.tsx:128]: Suggestion: you can simplify returning directly

```suggestion
			action: (subtask, mutate) => 
				testraySubTaskImpl.returnToOpen(subtask).then(() => 
					updateItemFromList(
						mutate,
						0,
						{},
						{
							revalidate: true,
						}
					);
				),
			
```

**Review by samuelleolopes — APPROVED** [review]
Good Job :fire:

**Review by kevenleone — APPROVED** [review]
LGTM 🔥

**Comment by kevenleone** [comment]
ci:forward

**Pull request closed without merging**